app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -45,9 +45,12 @@ def textual_bind_addr
   end
 
   def textual_server_state
+    server_state = translated_status(@record.properties['Calculated Server State'] || @record.properties['Server State'])
```

Comment:
What's the `Calculated server state` purpose, can we involve the suspend into the calculation? I would say that when something has a "calculated" in it's name, we should trust that the value is calculated form all inputs and reflect the state properly.

I would rather see the `Calculated Server State` changed then do this hackery below.

Reply:
The problem here is that the server state is the same when is suspended because server is running

Reply:
Sure, that's why there are three fields: `Server State`, `Suspend State` and `Calculated Server State`. And right now, I'm questioning the latest one, the `Calculated Server State`. It would be awesome to have`Calculated Server State` calculated right and reflecting the suspend...

Reply:
Yes that will be great, one "Final State" to show ^^

Reply:
Just in case you want to integrate the suspended state into the "Calculated" one, it's set here:
https://github.com/ManageIQ/manageiq-providers-hawkular/blob/888a77c9a2013bc9a25079da831e19afba22eb00/app/models/manageiq/providers/hawkular/inventory/parser/middleware_manager.rb#L196-L197

Reply:
Ok I read now this, I am gonna check, anyway finally....how we are gonna show this:
suspended
running ( suspended ) 
?

Reply:
As far as I can remember on the meeting we decided on _Suspended_

Reply:
I also remember we decided _Suspended_ to be consistent with other parts of MiQ.